internal-testutils/src/main/java/com/google/accompanist/internal/test/Assertions.kt

```diff
@@ -42,7 +42,7 @@ fun ImageBitmap.assertPixels(expected: Color, tolerance: Float = 0.001f) {
  * Run the [SemanticsNodeInteraction] provided by [block] repeatedly until either
  * the assertion succeeds, or the execution runs past [timeoutMillis].
  */
-fun SemanticsNodeInteraction.assertWithTimeout(
+public fun SemanticsNodeInteraction.assertWithTimeout(
```

Comment:
The change here is just updating visibilities, but I wonder if this utilities can be replaced with upstream replacements now.